main.py

```diff
@@ -117,6 +118,9 @@ def success_if(condition: Any) -> Tuple[TestResult, str]:
         return test_result(TestResult.NONZERO_EXIT_ERROR)
     except subprocess.TimeoutExpired:
         return test_result(TestResult.TIMEOUT_ERROR)
+    except:
```

Comment:
would be nice to capture the exception traceback as output for verbose mode. Something like this I suppose?

```suggestion
    except:
        output = traceback.format_exc()
```

(needs `import traceback` at the top, of course)